build/configs/qemu/README.md

```diff
@@ -75,3 +75,98 @@ Enter `Board Selection` -> `Build for qemu hardware`
 Select `NONE`(64KB) or `Build for SRAM increased QEMU Hardware`(16MB)  
 Enter `Chip Selection` -> `Boot Memory Configuration`  
 Set `Primary RAM size` to `65536`(64KB) or `16777216`(16MB)
+
+##Steps to verify network stack in QEMU is as follows
+build qemu as follow
+    a) cd os/tools
+    b) ./configure.sh qemu/tc_16m
+    c) cd ..
+    d) make
+
+Connect WiFi AP to Linux machine(wired connection. dhcp should be
+enabled in the AP. Then DHCP server will be running in AP)
+
+Create bridge and tap interface by adding below commands into shell
+script file(net_up.sh)
```

Comment:
can you specify the parameters here, for example net_up.sh tap0, so that one understands the uage well? Thanks.

Reply:
I have mentioned the usage of command on line no: 133.The above line just tells to add the below command into shell script. 
 execute the shell script with tap0 as argument(ex: sudo ./net_up.sh tap0)